src/AccessibilityInsights.Extensions.GitHubAutoUpdate/Extensions.GitHubAutoUpdate.csproj

```diff
@@ -40,6 +40,9 @@
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Newtonsoft.Json, Version=10.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
```

Comment:
Not sure why this reference was added?

Reply:
Code in this project consumes Newtonsoft, so this should have been here all along. Clean builds had build breaks about not having Newtonsoft, so I used VS to ensure that all Newtonsoft consumers were updated. VS inserted this reference as part of that process. It should stay

---
In reply to: [283034112](https://github.com/microsoft/accessibility-insights-windows/pull/353#discussion_r283034112) [](ancestors = 283034112)